package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "js-browserstack",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Let's add description, something like: Sample code repository to run selenium tests on `browserstack` using `NodeJs`